jekyll/_cci2/api-developers-guide.md

```diff
@@ -450,6 +450,184 @@ The following section details the steps you need to follow to download artifacts
+- `total credits used` - the total number of credits used
+- `windows_start & windows_end` - the time the window was first started, and then ended
+
+**Note** The above example only shows just a few builds. When you run this command, you may receive up to 250 individual builds that you can review in much more detail.
+
+### Reviewing Individual Build Metrics
+
+Now that you have retrieved aggregated data for up to 250 different builds, you will most likely want to review specific information about a single build, or smaller number of builds, to ensure that your builds are running efficiently. To review an individual build, follow the steps below.
+
+1. Using your project-slug from the previous API call you made to return workflow data, make a GET API call to the following insights endpoint:
+
+    ```sh
+    curl -X GET https://circleci.com/api/v2/insights/{project-slug}/workflows/builds
+    -H 'Accept: application/json'
+    ```
+4. Once you call this insights endpoint, you will receive a JSON output similar to the example shown below.
+
+```
+{
+  "items" : [ {
+    "id" : "08863cb6-3185-4c2f-a44e-b517b7f695a6",
+    "status" : "failed",
+    "duration" : 9263,
+    "created_at" : "2020-01-21T20:34:50.223Z",
+    "stopped_at" : "2020-01-21T23:09:13.953Z",
+    "credits_used" : 198981
+  }, {
+    "id" : "2705482b-40ae-47fd-9032-4113e976510f",
+    "status" : "failed",
+    "duration" : 9075,
+    "created_at" : "2020-01-21T20:14:00.247Z",
+    "stopped_at" : "2020-01-21T22:45:15.614Z",
+    "credits_used" : 148394
+  }, {
+    "id" : "65e049ee-5949-4c30-a5c6-9433ed83f96f",
+    "status" : "failed",
+    "duration" : 11697,
+    "created_at" : "2020-01-21T20:08:06.950Z",
+    "stopped_at" : "2020-01-21T23:23:04.390Z",
+    "credits_used" : 122255
+  }, {
+    "id" : "b7354945-32ee-4cb5-b8bf-a2f8c115b955",
+    "status" : "success",
+    "duration" : 9230,
+    "created_at" : "2020-01-21T19:31:11.081Z",
+    "stopped_at" : "2020-01-21T22:05:02.072Z",
+    "credits_used" : 195050
+  }, {
+    "id" : "7e843b39-d979-4152-9868-ba5dacebafc9",
+    "status" : "failed",
+    "duration" : 9441,
+    "created_at" : "2020-01-21T18:39:42.662Z",
+    "stopped_at" : "2020-01-21T21:17:04.417Z",
+    "credits_used" : 192854
+  }, {
+    "id" : "8d3ce265-e91e-48d5-bb3d-681cb0e748d7",
+    "status" : "failed",
+    "duration" : 9362,
+    "created_at" : "2020-01-21T18:38:28.225Z",
+    "stopped_at" : "2020-01-21T21:14:30.330Z",
+    "credits_used" : 194079
+  }, {
+    "id" : "188fcf84-4879-4dd3-8bf2-4f6ea724c692",
+    "status" : "failed",
+    "duration" : 8910,
+    "created_at" : "2020-01-20T03:09:50.448Z",
+    "stopped_at" : "2020-01-20T05:38:21.392Z",
+    "credits_used" : 193056
+  },
+```
+
+When reviewing each individual review build, please note that the following information is returned for each build, which is listed below.
```

Comment:
Think there's an extra word in here, also maybe we should be using pipeline run, rather than build? I'm not 100% sure.

Reply:
Fixed